.drone.yml

```diff
@@ -2,6 +2,10 @@ workspace:
   base: /go
 
 pipeline:
+  clone:
+    image: plugins/git
+    tags: true
```

Comment:
https://github.com/appleboy/drone-scp/blob/master/Makefile#L22-L26

Default clone behavior does not include all tags so you need to set `TAG` in environment section.

You can try it out by yourself.